README.md

```diff
@@ -58,20 +58,67 @@ pip install -U dataprep
 
 ## Examples & Usages
 
-The following examples can give you an impression of what dataprep can do:
+The following examples can give you an impression of what Dataprep can do:
 
 * [Documentation: Connector](https://sfu-db.github.io/dataprep/user_guide/connector/connector.html)
 * [Documentation: EDA](https://sfu-db.github.io/dataprep/user_guide/eda/introduction.html)
 * [EDA Case Study: Titanic](https://sfu-db.github.io/dataprep/user_guide/eda/titanic.html)
 * [EDA Case Study: House Price](https://sfu-db.github.io/dataprep/user_guide/eda/house_price.html)
 
+### Connector
+
+Connector is an intuitive, open-source API wrapper that speeds up development by standardizing calls to multiple APIs as a simple workflow.
+
+
+Connector provides a simple programming interface to collect structured data from different Web APIs (e.g., Twitter API, Yelp Fusion API, Spotify API, DBLP API), making web data collection easy and efficient, without requiring advanced programming skills.
```

Comment:
Instead of "programming interface" can we change it to "library"? Unless this is in reference to the GUI, in which case maybe "library and interface"?

Reply:
Done. "programming interface" was replaced by "wrapper".